install/helm/gloo/generate/values.go

```diff
@@ -308,7 +308,22 @@ type Gloo struct {
 }
 
 type KubeGateway struct {
-	Enabled *bool `json:"enabled,omitempty" desc:"Enable the Gloo Gateway Kubernetes Gateway API controller."`
+	Enabled           *bool                         `json:"enabled,omitempty" desc:"Enable the Gloo Gateway Kubernetes Gateway API controller."`
+	GatewayParameters map[string]*GatewayParameters `json:"gatewayParameters,omitempty" desc:"Map of GatewayClasses to default GatewayParameters"`
+}
+
+type GatewayParameters struct {
+	Image      *Image                 `json:"image,omitempty" desc:""`
+	Deployment *ProvisionedDeployment `json:"deployment,omitempty" desc:"Options specific to the deployment of the dynamically provisioned gateway proxy"`
+	Service    *ProvisionedService    `json:"service,omitempty" desc:"Options specific to the service of the dynamically provisioned gateway proxy"`
+}
+
+type ProvisionedDeployment struct {
+	Replicas *int64 `json:"replicas,omitempty" desc:"number of instances to deploy"`
```

Comment:
Let's add the default here?
```suggestion
	Replicas *int64 `json:"replicas,omitempty" desc:"The number of desired instances to deploy. Defaults to 1."`
```

Reply:
we shouldn't add defaults to the descriptions in values.go.
the helm docs generation gets the default values from values-template.yaml